frontend/src/organizations/components/OrganizationsTable.js

```diff
@@ -40,10 +40,18 @@ const OrganizationsTable = (props) => {
     id: 'inreserve',
     minWidth: 100
   }, {
-    accessor: item => Object.values(ORGANIZATION_STATUSES)
-      .find(element => element.id === item.status).description,
+    accessor: item => {
+      const orgStatus = Object.values(ORGANIZATION_STATUSES)
+        .find(element => element.id === item.status)
+      if (orgStatus.description === 'Active') {
+        return 'Yes'
+      } else if (orgStatus.description === 'Inactive') {
+        return 'No'
+      }
+      return orgStatus.description // default to the actual description if not Active/Inactive
+    },
```

Comment:
In the filter it is expecting to enter entire word "Yes/No", instead of filtering based on each character being entered.